### ISSUE TITLE
Linux 15kHz patch

### ISSUE THREAD

**Issue by D0023R** [Ansa89/linux-15khz-patch#1]
Hi Ansa89,
Thanks you very much for having maintained this project. 
Unfortunately, no updates have been made in your repository since May 7, 2017.
I am maintaining the 15kHz patches with additional changes required by recent kernels. I am planing to start a new repository for Linux kernel patches in the frame of groovymame new developments.
Do not hesitate to contact me.
Cheers!

**Comment by Ansa89** [comment]
Hi @D0023R,
thanks a lot for spending time maintaining those patches.
Unfortunately right now I don't have much time to keep it going (read: little to no time at all).
I can eventually help you with some minor things, but don't expect any "big activity" from my side (at least for the moment).

I can grant you read-write access to this repository, so you don't need to create another one, just tell me what do you prefer.

Sorry for the late reply and thanks again for your willingness to keep the patches alive.

**Comment by D0023R** [comment]
Hi @Ansa89,

Thank you for your support and openness for contributions. No worries, we all have constraints ;-)

I have already created another repository and addressed the 5.0.X and 5.1 kernel releases. I moved to a different directory structure which ease the management of the different releases. This structure might be use at some point by external (15khz tailored) distribution to automate the kernel patching.

Thanks again!
